patches/rrweb@2.0.0-alpha.13.patch

```diff
@@ -528,30 +528,248 @@ index 22fee601e786c1d8dfb5c01d2e359c8bcbac7c42..20c3e14adfde860563e8dd902041bd14
          let playbackRate = 1;
          if (typeof mediaAttributes.rr_mediaPlaybackRate === 'number') {
              playbackRate = mediaAttributes.rr_mediaPlaybackRate;
+diff --git a/es/rrweb/packages/rrweb-snapshot/es/css.js b/es/rrweb/packages/rrweb-snapshot/es/css.js
+new file mode 100644
+index 0000000000000000000000000000000000000000..34917db18e053a94d5c44ac857069ebb6133c8c1
+--- /dev/null
++++ b/es/rrweb/packages/rrweb-snapshot/es/css.js
+@@ -0,0 +1,87 @@
++import postcss from '../../../../../../../../../postcss/lib/postcss.js'
```

Comment:
Directory is after the patch is applied e.g. `node_modules/.pnpm/rrweb@2.0.0-alpha.13_patch_hash=6bgnomayryopibyfydha2z5wbi/node_modules/rrweb/es/rrweb/packages/rrweb-snapshot/es`